tests/cases/standalone/common/tql/basic.result

```diff
@@ -59,6 +59,28 @@ TQL EVAL (0, 10, '5s') test{k="a"};
 | 2.0 | 1970-01-01T00:00:10 | a |
 +-----+---------------------+---+
 
```

Comment:
Just to confirm if I need to add an sqlness test for `lookback` parameter?

Reply:
It would be great! Since `TQL EXPLAIN` would consider start and end, we can explain a few different lookback queries in sqlness

Reply:
hm... at the moment only `EVAL` has the `lookback` parameter. Do you mean it should be added for other commands as well?

Reply:
Yes 👍  `EVAL`, `ANALYZE` and `EXPLAIN` should have the same grammar.

Reply:
I've put some tests that support `lookback` grammar to address this comment. However, those tests have to be adjusted once the `lookback` is fully supported.